src/components/link/index.jsx

```diff
@@ -0,0 +1,47 @@
+import PropTypes from 'prop-types';
+
+const COMPONENT_CLASS_NAME = 'c-link';
+
+const ExternalLink = ({ children, href }) => (
+	<a
+		className={COMPONENT_CLASS_NAME}
+		href={href}
+		rel="noreferrer"
+		target="_blank"
+	>
+		<span className="visually-hidden">Opens in new window</span>
+		{children}
+	</a>
+);
+
+const Link = ({ children, href, isExternal }) => {
+	// internal link by default
+	if (isExternal) {
+		return (
+			<ExternalLink href={href}>
+				{children}
+			</ExternalLink>
+		);
+	}
+
+	return (
+		<a
+			className={COMPONENT_CLASS_NAME}
+			href={href}
+		>
+			{children}
+		</a>
+	);
+};
+
+Link.propTypes = {
+	children: PropTypes.node.isRequired,
+	href: PropTypes.string.isRequired,
+	isExternal: PropTypes.bool,
```

Comment:
We should have external link support built into the component by default by having a utility function to check if a link is external (starts with http/https) - Otherwise every consumer of this is going to have to validate if their link is internal or external
We could then add a prop that over rules our external logic for the cases where a consumer doesn't want to open in new window

Reply:
sounds good, can do

Reply:
updated